### PR TITLE
ConsoleExporter to print Activity(Span) Links

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Released 2021-Nov-29
   the `ConsoleMetricExporter` to export either manually or periodically.
   ([#2648](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2648))
 
+* ActivityExporter exports Activity Links.
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: nit: `ConsoleActivityExporter`

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs [hunk 2]
```diff
@@ -73,6 +73,15 @@ public override ExportResult Export(in Batch<Activity> batch)
                     }
                 }
 
+                if (activity.Links.Any())
+                {
+                    this.WriteLine("Activity.Links:");
+                    foreach (var activityLink in activity.Links)
+                    {
+                        this.WriteLine($"    {activityLink.Context.TraceId} {activityLink.Context.SpanId}");
```
Comment (utpilla) [inline src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs:81]: Do we not want to print the `Tags` for the activity link?

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md (hunk 1 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]:
```suggestion
* Added `Activity.Links` support to `ConsoleActivityExporter`.
```

Comment (reyang) [reply src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: This is supposed to be under "Unreleased" section?

**Review by reyang — CHANGES_REQUESTED** [review]
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2849/files#r798101161

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: fixed.

File: src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs (hunk 2 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs:81]: coming in future PRs.

**Review by reyang — APPROVED** [review]
LGTM with some suggestions.

File: src/OpenTelemetry.Exporter.Console/CHANGELOG.md (hunk 1 above)
Comment (reyang) [reply src/OpenTelemetry.Exporter.Console/CHANGELOG.md:15]: Reopen.

File: src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs (hunk 2 above)
Comment (cijothomas) [reply src/OpenTelemetry.Exporter.Console/ConsoleActivityExporter.cs:81]: So far I haven't seen any semantic convention using Links much... The 1st one I am seeing is https://github.com/open-telemetry/opentelemetry-specification/pull/2078/files It doesn't yet use tags.

**Pull request merged**